packages/block-editor/src/components/block-list/index.native.js

```diff
@@ -304,7 +304,9 @@ export class BlockList extends Component {
 					}
 					title={ title }
 					ListHeaderComponent={ header }
-					ListEmptyComponent={ ! isReadOnly && this.renderEmptyList }
+					ListEmptyComponent={
+						isRootList && ! isReadOnly && this.renderEmptyList
```

Comment:
We shouldn't be rendering this for inner blocks since it renders a blank space, this was breaking the new List V2 format. So now it will only render if it's the root list.